cmd/sidecar_mounter/main.go

```diff
@@ -34,9 +34,10 @@ import (
 )
 
 var (
-	gcsfusePath    = flag.String("gcsfuse-path", "/gcsfuse", "gcsfuse path")
-	volumeBasePath = flag.String("volume-base-path", webhook.SidecarContainerTmpVolumeMountPath+"/.volumes", "volume base path")
-	_              = flag.Int("grace-period", 0, "grace period for gcsfuse termination. This flag has been deprecated, has no effect and will be removed in the future.")
+	gcsfusePath           = flag.String("gcsfuse-path", "/gcsfuse", "gcsfuse path")
+	volumeBasePath        = flag.String("volume-base-path", webhook.SidecarContainerTmpVolumeMountPath+"/.volumes", "volume base path")
+	metricsScrapeInterval = flag.Int("metrics-scrape-interval", 10, "Scrape interval for gcsfuse metrics endpoint.")
```

Comment:
change the description to `Scrape interval in seconds for gcsfuse metrics endpoint.`

Reply:
Done.